lib/antora/extensions/antora-linked-worktree-patch.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+/* Copyright (c) 2022 OpenDevise, Inc.
```

Comment:
Sorry I missed this in your initial posting, but given this is hosted in Spring Security's code base can we please get the Copyright updated to be:

>  Copyright 2002-2022 the original author or authors.

Reply:
Not a problem.